src/main.py

```diff
@@ -310,6 +311,13 @@ def main():
                     break
 
                 stopwatch.reset()
+
+        except prawcore.exceptions.OAuthException as e_creds:
+            logging.error(e_creds)
+            logging.critical("Invalid login credentials. Check your .env!")
```

Comment:
Let's add a call to `traceback.print_exc()` to these handlers instead of logging `e_creds`, it does a more thorough job.